### ISSUE TITLE
[finetune] Dropout doesn't seem to affect training

### ISSUE THREAD

**Issue by olegklimov** [smallcloudai/refact#66]
dropout = 0.01, 0.1 gif animation
![dropout01](https://github.com/smallcloudai/refact/assets/18546213/85ad42cc-a972-45f0-ac45-0b0648fd69cd)

**Comment by JegernOUTT** [comment]
Can you try even higher values since you already have two runs to compare?
Quickly checked the code, haven't found any issues in parameter forwarding
